staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509.go

```diff
@@ -73,6 +73,10 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (user.Info, bool,
 
 	chains, err := req.TLS.PeerCertificates[0].Verify(optsCopy)
 	if err != nil {
+		switch err.(type) {
+		case x509.UnknownAuthorityError:
```

Comment:
The interface this stack returns is all wrong. It should be able to say one of {"yes", "hell no", and "I'm not sure"}. I think this condition represents an "I'm not sure" condition, since another authenticator might accept the cert.